vars/piperExecuteBin.groovy

```diff
@@ -156,6 +156,10 @@ void dockerWrapper(script, stepName, config, body) {
 
 // reused in sonarExecuteScan
 void credentialWrapper(config, List credentialInfo, body) {
+    if (config.containsKey('vaultAppRoleCredentialId') && config.containsKey('vaultAppRoleSecretCredentialId')) {
+        credentialInfo = [[type: 'token', id: 'vaultAppRoleCredentialId', env: ['PIPER_vaultApproleID']],
```

Comment:
I guess to be really correct we should use `vaultAppRoleIdCredentialsId` here...

Reply:
it's `vaultAppRoleTokenCredentialsId` now :)